Source/LudoEngine/Menu.cs

```diff
@@ -1,10 +1,100 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using System.Threading;
 
 namespace LudoEngine
 {
     public class Menu
     {
+
+        public static void PrintNoActivePieces()
+        {
+            Console.WriteLine();
+            Console.WriteLine("You need to roll 1 or 6 to move pieces from yard.");
+            Console.WriteLine();
+            Thread.Sleep(2000);
+        }
+
+        public static void PromtUserToRollDice()
+        {
+            Console.WriteLine();
+            Console.WriteLine("Press any key to roll the dice.");
+            Console.WriteLine();
```

Comment:
Använd /n och /r i consolewritline så slipper ni skriva tomma rader consolewritline så många gånger

Reply:
I C# räcker det väl med /n?

Reply:
https://stackoverflow.com/questions/5227698/carriage-return-and-line-feed-are-both-required-in-c/5227715